src/components/Accordion.tsx

```diff
@@ -98,11 +101,24 @@ export function Accordion<X extends Only<AccordionXss, X>>(props: AccordionProps
           ...xss,
         }}
         onClick={() => {
-          setExpanded(!expanded);
+          setExpanded((prior) => !prior);
           if (setExpandedIndex) setExpandedIndex(index);
         }}
       >
-        <span css={Css.fg1.tl.$}>{title}</span>
+        {titleOnClick ? (
+          <button
```

Comment:
Suggestion: We should try to avoid nesting interactive elements. Per HTML standards, nesting of interactive elements is not permitted- it's possible, just not advisable. 
> Content model:
> Phrasing content, but there must be no interactive content descendant and no descendant with the tabindex attribute specified.

https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element

Please work with design to determine a way forward where we do not nest interactive elements. We could have buttons side by side, or just make the chevron the click target for expanding/collapsing. 

I used to be pretty adamant with the old design crew about avoiding these patterns, and we've been successful so far. I'd like us to keep up with enforcing HTML standards back onto design and help educate them. 

Fwiw, here's what GPT had to say about it:

Nesting a `<button>` element inside another `<button>` element is invalid HTML according to the HTML5 specification. This is because:

1. **Semantic Incorrectness**: Buttons are meant to be interactive elements, and nesting one interactive element inside another can create confusion about which element should be interactive.

2. **Accessibility Issues**: Screen readers and other assistive technologies may have difficulty interpreting nested buttons, leading to poor user experiences for individuals relying on these tools.

3. **Browser Compatibility**: Different browsers may handle nested buttons inconsistently. Some browsers may ignore the nested button, while others may render it in unexpected ways, leading to unpredictable behavior.

4. **Event Handling Confusion**: Nesting buttons can create complications in event handling. For instance, click events might not propagate as expected, causing issues in the functionality of the nested buttons.

To maintain valid, accessible, and predictable HTML, it's best to avoid nesting `<button>` elements inside each other.